wolfssl/test.h

```diff
@@ -64,7 +64,6 @@
 
     static int wolfssl_tcp_select(int sd, int timeout)
```

Comment:
It looks like you can remove this `wolfssl_tcp_select` function all-together? Its now in `IDE/MDK-ARM/MDK-ARM/wolfSSL/wolfssl_MDK_ARM.c`.

Reply:
Note that CMSIS pack works with MDK5, instead of MDK4 under MDK-ARM.
Please review changes around wolfssl_tcp_select, 98f85c9.